src/lightning/fabric/CHANGELOG.md

```diff
@@ -30,6 +30,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added a warning when calling methods on `_FabricModule` that bypass the strategy-specific wrappers ([#17424](https://github.com/Lightning-AI/lightning/pull/17424))
 
 
+- Added `Fabric.init_module()` context manager to instantiate large models efficiently ([#17462](https://github.com/Lightning-AI/lightning/pull/17462))
+
+
+- Added `lightning.fabric.plugins.Precision.module_init_context()` context manager to control the default tensor dtype during model instantiation ([#17462](https://github.com/Lightning-AI/lightning/pull/17462))
+
+
+- Added `lightning.fabric.strategies.Strategy.module_init_context()` context manager to control the model instantiation ([#17462](https://github.com/Lightning-AI/lightning/pull/17462))
```

Comment:
Same suggestion. This should mention that it'll use the meta device etc

Reply:
Ok, I added a more verbose description with two bullet points